### DISCUSSION TITLE
[User Question]

### DISCUSSION THREAD

**Discussion by NeonGooRoo** [vuepress-theme-hope/vuepress-theme-hope#4135] [Q&A] [answered]
### Checklist

- [X] I searched existing issues and no one else requests similar feature.
- [X] I think that 25%+ users are positive towards this feature.

### Describe the feature

Add info into the guide on how to add media links like the one for Github and the one you have for Telegram.
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/123720895/fb6da20e-db34-4b80-9796-017b242bfd26)
I can not find any information on how to do that. I can edit the github repo link, it work, but how do I add my telegram or maybe other media

**Comment by Mister-Hope** [comment, marked as answer]
https://theme-hope.vuejs.press/guide/layout/navbar.html#layout-config
